lib/p2p/Peer.ts

```diff
@@ -126,7 +130,7 @@ class Peer extends EventEmitter {
       case 'LTC':
         return this.handshakeState.lndltcPubKey;
       default:
-        return;
+        return undefined;
```

Comment:
I think the `undefined` is implied  with a blank `return;` statement. I'd leave it as is unless you think it's more clear this way or otherwise better.

Reply:
It just seemed more aesthetic to me, but I think you are right.